tomcat-base/src/shared/java/eu/xenit/alfresco/tomcat/embedded/Main.java

```diff
@@ -35,8 +35,12 @@ public static void main(String[] args) {
         }
     }
 
-    private static void setGlobalProperties(Configuration configuration) {
-        configuration.getGlobalProperties().forEach(System::setProperty);
+    private static void setSystemProperties(Configuration configuration) {
+        configuration.getSystemProperties().forEach((key, value) -> {
+            if (System.getProperty(key) == null) {
```

Comment:
What's the reason for adding a null check on the value here, when the previous impl did not?

Reply:
Now that you mention it. This used to be a bug. JAVA_OPTS would have been overridden by defaults in java.

Reply:
I think I need to add some tests.